api/lib/opentelemetry.rb

```diff
@@ -28,7 +28,11 @@ module OpenTelemetry
 
   # @return [Object, Logger] configured Logger or a default STDOUT Logger.
   def logger
-    @logger ||= Logger.new($stdout, level: ENV['OTEL_LOG_LEVEL'] || Logger::INFO)
+    @logger ||= create_logger
```

Comment:
These changes are directly related to the Logs instrumentation. They need a PR when the logs instrumentation is ready for review.